modules/test/playwright/tests/frontend-data-set-views-web/selectionFilters.spec.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import getRandomString from '../../utils/getRandomString';
+import {dataSetManagerApiHelpersTest} from './fixtures/dataSetManagerApiHelpersTest';
+import {dataSetManagerSetupTest} from './fixtures/dataSetManagerSetupTest';
+import {filtersPageTest} from './fixtures/filtersPageTest';
+import {picklistApiHelpersTest} from './fixtures/picklistApiHelpersTest';
+
+const SELECTION_FILTER_NAME = 'Selection filter';
+const PICKLIST_VALUE_KEY = 'sampleValue';
+const PICKLIST_VALUE_NAME = 'Sample Value';
+
+export const test = mergeTests(
+	dataSetManagerApiHelpersTest,
+	featureFlagsTest({
+		'LPD-10754': true,
```

Comment:
This FF no longer exists.

Reply:
You can remove this line. It has no effect, apart from making an unnecessary and maybe failed request in the fixture.

Reply:
Oh, my bad, I was thinking of `LPD-10735`, please ignore my comment. This is fine :+1: